src/resources/views/crud/fields/checklist.blade.php

```diff
@@ -25,11 +25,19 @@
 
 @include('crud::fields.inc.wrapper_start')
     <label>{!! $field['label'] !!}</label>
+    <div class="row">
+        <div class="col-sm-4">
+            <label class="font-weight-normal">
+                <input type="checkbox"  id="select-all" {{ count($field['options']) == count($field['value']) ? 'checked' : '' }}> 
```

Comment:
How does this look with a checkbox @kamrul-bd ? Do you have a screenshot? I'm thinking out loud here... any reason to make this a checkbox and not a simple anchor with a link? After all... it won't hold any value, it's just used as a trigger.